app/src/main/java/de/lukasneugebauer/nextcloudcookbook/recipe/util/RecipeCreateEditViewModel.kt

```
@@ -108,6 +115,59 @@ abstract class RecipeCreateEditViewModel(
+                        } else {
+                            _imageUploadError.value = UiText.StringResource(R.string.error_image_upload_failed)
+                        }
+                    }
+                    is Resource.Error -> {
+                        _imageUploadError.value =
+                            result.message ?: UiText.StringResource(R.string.error_image_upload_failed)
+                    }
+                }
+            } catch (e: kotlinx.coroutines.CancellationException) {
+                throw e
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)